<?xml version="1.0"?>
<launch>
  <param name="robot_description" command="cat '$(find kurt3d)/urdf/3d_scanner.urdf'" />

  <node pkg="robot_state_publisher" type="state_publisher" name="state_publisher" />
  <node pkg="kurt3d" type="laserscanner_node" name="laserscanner_node" />
  <node pkg="kurt3d" type="servo_node" name="servo_node" />



  <include file="$(find kurt_bringup)/launch/sick_lms200.launch">
    <param name="resolution" value="0.5" />
  </include>


    <node type="laser_scan_assembler" pkg="laser_assembler" name="scan_assembler">
    <remap from="scan" to="cleaned_scan" />
    <param name="max_scans" type="int" value="5000" />
    <param name="fixed_frame" type="string" value="base_link" />
   </node>

</launch>
